geode-core/src/main/java/org/apache/geode/examples/SimpleSecurityManager.java

```diff
@@ -39,7 +40,11 @@ public void init(final Properties securityProps) {
   public Object authenticate(final Properties credentials) throws AuthenticationFailedException {
     String token = credentials.getProperty(TOKEN);
     if (token != null) {
-      return "Bearer " + token;
+      if (VALID_TOKEN.equals(token)) {
```

Comment:
Is this class really example code (as the package implies) or is it test code (as the class comment states)? If it is example code then consider adding some clarifying comments. If it is test code can it be moved to another package?